plugins/woocommerce-admin/client/core-profiler/services/installAndActivatePlugins.ts

```diff
@@ -188,7 +194,18 @@ export const pluginInstallerMachine = createMachine(
 			} ),
 			assignPluginsInstallationQueue: assign( {
 				pluginsInstallationQueue: ( ctx ) => {
-					return ctx.selectedPlugins;
+					return ctx.selectedPlugins.slice().sort( ( a, b ) => {
```

Comment:
I'd love it if there's a comment on the sorting that is done here, i.e: ascending or descending